asciidoc-java/src/test/java/io/yupiik/asciidoc/parser/ParserTest.java

```diff
@@ -131,6 +131,13 @@ void parseHeader() {
         assertEquals(Map.of("attr-1", "v1", "attr-2", "v2"), header.attributes());
     }
 
+    @Test
+    void parseHeaderWithoutTitle() {
```

Comment:
Is this actually valid? I would have rather throw an exception if trying to parse a header without a header 🤔

Reply:
Have a look to the link I provided in the desc, https://docs.asciidoctor.org/asciidoc/latest/document/header/, the title is optional..